DER/ExchangeTradedDerivatives/Futures.rdf

```diff
@@ -540,13 +386,6 @@
 		<rdfs:range rdf:resource="&fibo-der-etd-std;DerivativesPriceDeterminationMethod"/>
 	</owl:ObjectProperty>
 	
-	<owl:DatatypeProperty rdf:about="&fibo-der-etd-fut;securityDescription">
-		<rdfs:label xml:lang="en">security description</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-etd-fut;FutureInstrument"/>
-		<rdfs:range rdf:resource="&xsd;string"/>
-		<skos:definition xml:lang="en">The name of the security that uniquely identifies it amongst all other securities.</skos:definition>
-	</owl:DatatypeProperty>
-	
 	<owl:ObjectProperty rdf:about="&fibo-der-etd-fut;specifiesDeliveryMethod">
```

Comment:
Unclear why we need this - at minimum it should be a sub property of fbc-ftc-ra;specifies and the range should be SecurityDeliveryMethod or a subclass

Reply:
Removed the property, used fibo-fbc-fct-ra;specifies in the restriction, retained the range since the DeliveryTerms came from DerivativesBasics and may include the delivery method but other things as well.

Reply:
Also made the remaining standardized terms children of StandardizedTerms from FinancialInstruments.